source/features/bypass-checks-travis.js

```diff
@@ -0,0 +1,48 @@
+import select from 'select-dom';
+import domify from '../libs/domify';
+
+export default () => {
+	const checks = select.all('.merge-status-item');
+	const checkTravisBranch = checks.find(isTravisBranchCheck);
+	const checkTravisPR = checks.find(isTravisPRCheck);
+
+	if (checkTravisBranch) {
+		bypassCheck(checkTravisBranch);
+	}
+	if (checkTravisPR) {
+		bypassCheck(checkTravisPR);
+	}
+};
+
+async function bypassCheck(check) {
+	const details = select('.status-actions', check);
+	const checkUrl = details.getAttribute('href');
+
+	const response = await fetch(checkUrl, {
+		credentials: 'include'
+	});
+
+	if (response.ok) {
+		const dom = domify(await response.text());
+		const links = select.all('[id^="check_run_"] a', dom);
+		const viewMoreLink = links.find(isViewMoreLink);
+
+		if (viewMoreLink) {
+			details.setAttribute('href', viewMoreLink.getAttribute('href'));
+			details.setAttribute('rel', viewMoreLink.getAttribute('rel'));
+			details.setAttribute('target', viewMoreLink.getAttribute('target'));
```

Comment:
All of these are properties as well, I think, so you can use `details.rel = viewMoreLink.rel`

Reply:
Also is this the behavior we want? Should it _replace_ the previous link?

Reply:
I always prefered to access/set attributes with `getAttribute`/`setAttribute` , but here I think we can directly use properties yes.

For the behavior, this was my first suggestion in #1620.
I think it's not a problem to totally replace the link `Details` , no one is actually crazy enough to see Travis checks in `Checks` tab.

Reply:
> The suggestion to use props stands

Updated.